standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/txn/CallProperties.java

```diff
@@ -0,0 +1,112 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.hadoop.hive.metastore.txn;
+
+import org.apache.hadoop.hive.metastore.api.MetaException;
+import org.datanucleus.transaction.TransactionIsolation;
+import org.springframework.transaction.PlatformTransactionManager;
+
+import java.sql.Connection;
+import java.util.function.Function;
+
+public class CallProperties {
```

Comment:
What is the intention of that class?

Reply:
This class holds the attributes of the retry call: error handler, transaction isolation level, rollback-on-error, etc. Using this class  there is no need to pass all the values as a separate parameter. Also, by setting the default values to the most common one, typically only 1-2 properties have to be set for a particular retry-call.

Reply:
Ok, thx